compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/CallableReferenceLowering.kt

```diff
@@ -158,6 +146,12 @@ internal class CallableReferenceLowering(val context: JvmBackendContext) : FileL
         })
     }
 
+    private val arrayGetFun =
```

Comment:
@pyos Please move these declaration into FunctionReferenceBuilder where they are used: this will allow to avoid overhead with loading lazy data when it is not necessery.

Reply:
@max-kammerer done